### PR TITLE
support neovim float window

### PR DESCRIPTION
**Pull request by skt041959**
By using float window, it can be used on a read only buffer or on a terminal buffer

### CODE REVIEW

**Comment by globalchubby** [comment]
Cool, thanks for the contribution!

Can you post a video showing how this can be used? And also update the help documentation?

File: autoload/snipe/core.vim [hunk 1]
```diff
@@ -153,6 +153,45 @@ function! s:SafeSetLine(lnum, line) " {{{
   call setline(a:lnum, a:line)
   redraw
 endfunction
+
+let s:buf = 0
```
Comment (globalchubby) [inline autoload/snipe/core.vim:157]: Is there a way to make this not global, and instead pass it around as a parameter?

**Comment by skt041959** [comment]
Hi,
I didn't change any of the usage or keymap.
It doesn't need any other configuration. When the neovim floating window feature is detected, it will be enabled automatically. 
There is an option `let g:snipe_float_window=0` to disable this feature.

File: autoload/snipe/core.vim (hunk 1 above)
Comment (skt041959) [reply autoload/snipe/core.vim:157]: I can't find a way to do that. I make it global to avoid creating a new buffer every time.

**Pull request merged**